environment.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   - adjusttext=0.7.3.1
   - bioconda::snakemake-minimal=8.17.0
   - docker-py=5.0
+  - podman=5.4.0
```

Comment:
should this also be 5.4.0.1?

Reply:
I ran into an issue when I pinned it to 5.4.0.1 and ignored it - I could look more into it, though I'll circle back to this PR since I figured out what was blocking me in the NetMix2 integration.